server/src/routes/api/v1/recipes/recipe.route.ts

```diff
@@ -17,14 +17,14 @@ export class RecipeRoute {
     this._router.post("/",
       this._authValidationHandler.checkToken,
       this._authValidationHandler.checkUser,
-      this._authValidationHandler.nonRestricted,
+      //this._authValidationHandler.nonRestricted,
       this._modelValidationHandler.validate,
       this._recipeHandler.save);
 
     this._router.get("/:id",
       this._authValidationHandler.checkToken,
-      this._authValidationHandler.checkUser,
```

Comment:
I have added an approach to make the request itself extendable, so we don't have to remove the properties and they don't pollute the body